test/helpers/browser.js

```diff
@@ -8,7 +8,8 @@ var vm = require('vm');
 function createContext() {
   var context = {
     // required by browserify
-    XMLHttpRequest: function() { throw new Error('not implemented'); },
```

Comment:
Is this change also associated with the upgrade?

Reply:
Yes. Otherwise, it throws an error - not implemented